recipe/README.md

```diff
@@ -11,6 +11,9 @@ More information for different classes of users is documented in the documents b
 
 ## Versioning
 
+On conda-forge, the version of installed CUDA packages can be controlled by
```

Comment:
The nvidia channel also uses cuda-version in 12.5+. Should that be noted?

Reply:
It was noted elsewhere, but I don't mind adding it here. How about this?
```suggestion
On the conda-forge channel (and the nvidia channel since CUDA 12.5+), the
version of installed CUDA packages can be controlled by
```

Reply:
Sounds good!